src/workflows/dags/airqo_measurements.py

```
@@ -474,7 +479,9 @@ def send_hourly_measurements_to_bigquery(airqo_data: pd.DataFrame):
         from airqo_etl_utils.bigquery_api import BigQueryApi
         from airqo_etl_utils.airqo_utils import AirQoDataUtils
 
-        data = AirQoDataUtils.process_aggregated_data_for_bigquery(data=airqo_data)
+        data = DataUtils.format_data_for_bigquery(
+            data, DataType.AVERAGED, DeviceCategory.GENERAL, Frequency.HOURLY
```

Ruff (0.8.2)

Undefined name `data`

(F821)